### PR TITLE
docs: add basic intro page for C#

### PR DESCRIPTION
**Pull request by yury-s**
* Drive-by: update default docker image refs to focal.

### CODE REVIEW

File: docs/src/intro-csharp.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Command Line Interface [CLI](./cli.md) can be used to record user interactions and generate C# code.
+
+```sh
+# FIXME:
+```
+
+## System requirements
+
+Playwright requires Python version 3.7 or above. The browser binaries for Chromium,
```
Comment (aslushnikov) [inline docs/src/intro-csharp.md:51]: python!

File: docs/src/intro-java.md [hunk 2]
```diff
@@ -133,5 +133,5 @@ Firefox and WebKit work across the 3 platforms (Windows, macOS, Linux):
 * **Linux**: Depending on your Linux distribution, you might need to install additional
   dependencies to run the browsers.
   * Firefox requires Ubuntu 18.04+
-  * For Ubuntu 18.04, the additional dependencies are defined in [our Docker image](https://github.com/microsoft/playwright/blob/master/utils/docker/Dockerfile.bionic),
+  * For Ubuntu 18.04, the additional dependencies are defined in [our Docker image](https://github.com/microsoft/playwright/blob/master/utils/docker/Dockerfile.focal),
```
Comment (aslushnikov) [inline docs/src/intro-java.md:136]: ubuntu 18.04 yet the docker is focal

File: docs/src/intro-js.md [hunk 3]
```diff
@@ -105,5 +105,5 @@ Firefox and WebKit work across the 3 platforms (Windows, macOS, Linux):
 * **Linux**: Depending on your Linux distribution, you might need to install additional
   dependencies to run the browsers.
   * Firefox requires Ubuntu 18.04+
-  * For Ubuntu 18.04, the additional dependencies are defined in [our Docker image](https://github.com/microsoft/playwright/blob/master/utils/docker/Dockerfile.bionic),
+  * For Ubuntu 18.04, the additional dependencies are defined in [our Docker image](https://github.com/microsoft/playwright/blob/master/utils/docker/Dockerfile.focal),
```
Comment (aslushnikov) [inline docs/src/intro-js.md:108]: ditto

**Review by aslushnikov — APPROVED** [review]

File: docs/src/intro-csharp.md (hunk 1 above)
Comment (yury-s) [reply docs/src/intro-csharp.md]: Removed.

File: docs/src/intro-java.md (hunk 2 above)
Comment (yury-s) [reply docs/src/intro-java.md]: Updated.

**Pull request merged**